WTWSMS/events/religious_events.txt

```diff
@@ -3763,11 +3809,16 @@ character_event = {
 			
 			# Pragmatism
 			modifier = {
-				factor = 3  # Powerful rulers want a modern religion
+				factor = 4  # Powerful rulers want a modern religion
```

Comment:
They are pragmatic, and therefore the number is 3 rather than 4, given that a too high one will lead to zealot behaviour.